project_template/CMakeLists.txt

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required(VERSION 3.15)
+
+project(az_exampleshortname
+    LANGUAGES C CXX
+    VERSION 0.1.0)
+
+option(az_exampleshortname_with_docs "Build documentation" ON)
+option(BUILD_TESTING "Build tests" ON)
+
+
+
+
+
+# this is a cmake 3.15 feature that tells find_package to 

```

Comment:
I can't tell what "this" is referring to here. (Is it the enormous block of empty lines above? There must be *some* reason for all those empty lines.)